src/message.ts

```diff
@@ -12,5 +12,11 @@ export function warn(message: unknown, ...args: unknown[]): void {
 }
 
 function prefix(message: unknown): string {
-  return chalk.magentaBright`(Λ) ` + (typeof message === "string" ? message : inspect(message));
+  return (
+    "pid:" +
+    process.pid +
```

Comment:
This seems like leftover debugging? It's not very user-friendly.

Reply:
I added this with a temporary commit for @petecheslock to check to be sure. I'll remove it now.